pkgs/by-name/cu/cue/package.nix

```diff
@@ -24,7 +24,7 @@ buildGoModule rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
-  ldflags = [ "-s" "-w" "-X cuelang.org/go/cmd/cue/cmd.version=${version}" ];
+  ldflags = [ "-s" "-w" "-X cuelang.org/go/cmd/cue/cmd.version=v${version}" ];
```

Comment:
```suggestion
  ldflags = [ "-s" "-w" ];
```

Reply:
And add `version = "v${version}";` to `testVersion`.

Reply:
fixed!